internal/node/component/info/handler_node.go

```diff
@@ -18,25 +32,317 @@ type Version struct {
 	Commit string `json:"commit"`
 }
 
-// GetNodeOperator returns the node information.
-func (c *Component) GetNodeOperator(ctx echo.Context) error {
+type NodeInfoResponse struct {
+	Data NodeInfo `json:"data"`
+}
+
+type GIRewardsResponse struct {
+	Data []struct {
+		ID                    uint64 `json:"id"`
+		TotalOperationRewards string `json:"total_operation_rewards"`
+		TotalStakingRewards   string `json:"total_staking_rewards"`
+		TotalRequestCounts    string `json:"total_request_counts"`
+	} `json:"data"`
+}
+
+type GINodeInfoResponse struct {
+	Data struct {
+		LastHeartbeat int64  `json:"last_heartbeat"`
+		SlashedTokens string `json:"slashed_tokens"`
+	} `json:"data"`
+}
+
+type NodeInfo struct {
+	Operator   common.Address `json:"operator"`
+	Version    Version        `json:"version"`
+	Uptime     int64          `json:"uptime"`
+	Parameters string         `json:"parameters"`
+	Coverage   float64        `json:"coverage"`
+	Records    Record         `json:"records"`
+}
+
+type Record struct {
+	LastHeartbeat  int64    `json:"last_heartbeat"`
+	RecentRequests []string `json:"recent_requests"`
+	RecentRewards  []Reward `json:"recent_rewards"`
+	SlashedTokens  string   `json:"slashed_tokens"`
+}
+
+type Reward struct {
+	Epoch            uint64 `json:"epoch"`
+	OperationRewards string `json:"operation_rewards"`
+	StakingRewards   string `json:"staking_rewards"`
+	RequestCounts    string `json:"request_counts"`
+}
+
+// GetNodeInfo returns the node information.
+func (c *Component) GetNodeInfo(ctx echo.Context) error {
 	go c.CollectMetric(ctx.Request().Context(), ctx.Request().RequestURI, "info")
 
 	zap.L().Debug("get node info", zap.String("request.ip", ctx.Request().RemoteAddr))
 
-	response := fmt.Sprintf("This is an RSS3 Node operated by %s.", c.config.Discovery.Operator.EvmAddress)
+	// Get Version info
+	version := c.buildVersion()
+
+	// Get Operator address info
+	evmAddress := common.Address{}
+
+	if !lo.IsEmpty(c.config.Discovery.Operator.EvmAddress) {
+		evmAddress = c.config.Discovery.Operator.EvmAddress
+	}
```

Comment:
```suggestion
	// Get Operator address info
	evmAddress := c.config.Discovery.Operator.EvmAddress
```

Reply:
if user don't set operator evm address in config, program will panic if using `c.config.Discovery.Operator.EvmAddress` directly.